app/mailers/user_mailer.rb

```diff
@@ -92,7 +92,7 @@ def subject_for(key)
       default: [:subject, key.to_s.humanize], app_name: app_name)
   end
 
-  def preview?
-    GovukAdminTemplate.environment_label == "Preview"
+  def integration?
+    Plek.environment == "integration"
```

Comment:
Makes sense. One option as we'd discussed would be the `defined?(ActiveSupport)` dance or maybe we could just place the string comparison checks in one canonical predicate in this app? So at least then we don't repeat the `Plek.environment == "..."` things in various spots.